# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/socket.h.patch
# Copyright (C) 2011 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

fix udev: libudev-monitor.c:157:52: error: 'SO_RCVBUFFORCE' undeclared (first use in this function)

  - Rene Rebe <rene@exactcode.de>

--- dietlibc/include/sys/socket.h.vanilla	2011-01-05 14:48:32.000000000 +0100
+++ dietlibc/include/sys/socket.h	2011-01-05 14:49:11.000000000 +0100
@@ -145,6 +145,8 @@
 
 #endif
 
+#define SO_RCVBUFFORCE  33
+
 #if !defined(__hppa__) && !defined(__sparc__)
 /* Security levels - as per NRL IPv6 - don't actually do anything */
 #define SO_SECURITY_AUTHENTICATION		22
